gate-ldap/src/main/groovy/com/netflix/spinnaker/gate/security/ldap/LdapSsoConfig.groovy

```diff
@@ -86,8 +87,12 @@ class LdapSsoConfig extends WebSecurityConfigurerAdapter {
     }
   }
 
+  @Autowired
+  DefaultCookieSerializer defaultCookieSerializer
```

Comment:
This would be better before method definitions start, next to the other autowired properties.